polynote-frontend/polynote/messages.js

```diff
@@ -80,6 +80,38 @@ export class LoadNotebook extends Message {
 
 LoadNotebook.codec = combined(shortStr).to(LoadNotebook);
 
+export class DownloadNotebook extends Message {
```

Comment:
Hmm... the upload/download functionality seems better suited to REST operations than the binary protocol to me.

Reply:
I guess you could make the same argument for create/delete

Reply:
🤦‍♂️ remember how I told you on Slack that "some code isn’t necessary because I started down a different path"? When I looked at it again today I totally forgot which code that was and totally missed it! I'd figured that maybe I'd already deleted the code I was talking about. In fact... it's this. 🤦‍♂️🤦‍♂️🤦‍♂️

All the `DownloadNotebook` and `NotebookFile` stuff are all implemented but not actually used! I'm using the `download=true` query and the modification to `CreateNotebook` for all of the import/export stuff. Let me clean this up real quick... 😩😩😩

Reply:
All cleaned up now! A lot less code now lol